server/mailer/templates/registrationTemplate.ejs

```diff
@@ -0,0 +1,345 @@
+`<!DOCTYPE html>
```

Comment:
Is this opening tick-mark intentional?

```suggestion
<!DOCTYPE html>
```

Reply:
[bc9cbcf](https://github.com/AMP-SCZ/dpdash/pull/457/commits/bc9cbcfb5650ebceec9074f06d4e04dcc47fc58f)